app/components/form-block-radiobutton/style.scss

```diff
@@ -0,0 +1,29 @@
+& {
+  cursor: pointer;
+  padding: 18px;
+  margin: 25px;
+  background: #EEF2F3;
```

Comment:
This one should also be a variable. It's used in preprints in a couple of places.

Reply:
Thoughts on switching to `#EEEEEE`, it's a `$color-light` in osf-style?

Reply:
`#EEF2F3` is a little more blue than `#EEEEEE` which looks better with the background color, `#FCFCFC`.
![screen shot 2017-08-28 at 11 45 51 am](https://user-images.githubusercontent.com/7131985/29781218-7a9a72a4-8be6-11e7-81e4-9f5316db3e48.png)

Reply:
👍